x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_integrate.test.tsx

```diff
@@ -9,9 +9,8 @@ import '../../../__mocks__/shallow_useeffect.mock';
 
 import React from 'react';
 
-import { mount } from 'enzyme';
-
 import { EuiCodeBlock, EuiDescriptionListDescription } from '@elastic/eui';
+import { mountWithIntl } from '@kbn/test-jest-helpers';
```

Comment:
This was updated as we had a console error exception in the tests when it was changed to `FormattedMessage` complaining about intl not being available.